client/client.go

```diff
@@ -188,5 +188,9 @@ func (c *Client) GetLedgerChannel(id types.Destination) (query.LedgerChannelInfo
 
 // Close stops the client from responding to any input.
 func (c *Client) Close() error {
+	err := c.store.Close()
```

Comment:
Out of scope of this PR: I just want to flag the slight discrepancy between the relationship between `client`, `store` and `engine` here versus in the organisation of our packages. 

In the latter, the `store` belongs to the `engine`. Here, it belongs to the `client`.

Reply:
That's a good point! I think the `client` gained access to the `store` when we introduced the query API. The `client` can directly query the `store` instead of having to go through the engine to look things up. We should probably be consistent and have either the `client` or `engine` "own" the `store` and be responsible for closing it.

Reply:
Yep. Probably the `store` should be a sibling of `engine`, not a child. But likely more pain than it is worth to fix.